charts/spegel/values.yaml

```diff
@@ -137,6 +137,18 @@ grafanaDashboard:
   # See: sidecar.dashboards.folderAnnotation for specifying the dashboard folder.
   # https://github.com/grafana/helm-charts/tree/main/charts/grafana
   annotations: {}
+  grafanaOperator:
```

Comment:
Let me know if you think we should make this a top level `grafanaOperatorDashboard` and adjust the logic to suite it. I nested it for simplicity and they are related.